src/js/angular/ttyg/templates/chat-view.html

```diff
@@ -0,0 +1,28 @@
+<link href="css/ttyg/chat-view.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
+
+<div class="chat-view">
+    <div class="messages" ng-class="{'hidden': !selectedChat}">
```

Comment:
I see that you just copy-pasted the template, but it doesn't make any sence to just move all the crappy legacy code around without trying to improve it a bit.
Here, this `selectedChat` doesn't make sence. It might have been meaningful in the context of the view template from where you extracted it, but here it should be just `chat` IMO

Reply:
Renamed